aten/src/ATen/native/native_functions.yaml

```diff
@@ -105,3 +106,14 @@
   dispatch:
     CPU: RoiPooling2d_backward_cpu
     CUDA: RoiPooling2d_backward_cuda
+
+- func: _convolution_nogroup(Tensor input, Tensor weight, Tensor? bias, IntList stride, IntList padding, IntList dilation, bool transposed, IntList output_padding) -> Tensor
+  variants: function
+
+- func: _convolution(Tensor input, Tensor weight, Tensor? bias, IntList stride, IntList padding, IntList dilation, bool transposed, IntList output_padding, int64_t groups, bool benchmark, bool deterministic, bool cudnn_enabled) -> Tensor
+  variants: function
+
+# NB: We MUST call the input self, otherwise codegen will attempt to
```

Comment:
I think our current invariant is that all grads are always defined

Reply:
Definitely not true today, but it's something that we are interested in fixing, it seems...